core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -1172,7 +1215,9 @@ ORDER BY nonce ASC
 		if err = orm.LoadTxesAttempts(ctx, etxs); err != nil {
 			return pkgerrors.Wrap(err, "FindTransactionsConfirmedInBlockRange failed to load evm.tx_attempts")
 		}
-		err = orm.loadEthTxesAttemptsReceipts(ctx, etxs)
+
+		// retrieve tx with attempts which contain partial attempts values for optimization purpose
```

Comment:
nit:
```suggestion
		// retrieve tx with attempts which contain partial receipts values for optimization purpose
```

Reply:
Same as the other method I think it would be useful to add a description above the function and in the interface that also calls out that the receipts are partially populated